docs/ek_certificates.md

```diff
@@ -0,0 +1,102 @@
+# Research of EK certificate chain sizes
+
+This document combines results gathered from [this issue](https://github.com/fobnail/docs/issues/50).
+It was created to set expectations about size of data that has to be sent to and
+parsed by Fobnail Token.
+
+## SLB 9665TT2.0 TPM
+
+Used on PC Engines apu platforms:
+
+```text
+Certificate 0 size: 1177 bytes
+Certificate 1 size: 1463 bytes
+Certificate 2 size: 1455 bytes
+Certificate is self-signed, assuming it is root
+
+Chain length: 3
+Total chain size: 4095 bytes
+Size of the biggest certificate: 1463 bytes
+```
+
+## SLB 9670
+
+Used on Raspberry Pi. Two different specimens had slightly different chains:
+
+```text
+Certificate 0 size: 1422 bytes
+Certificate 1 size: 1705 bytes
+Certificate 2 size: 1455 bytes
+Certificate is self-signed, assuming it is root
+
+Chain length: 3
+Total chain size: 4582 bytes
+Size of the biggest certificate: 1705 bytes
+```
+
+```text
+Certificate 0 size: 1171 bytes
+Certificate 1 size: 1449 bytes
+Certificate 2 size: 1455 bytes
+Certificate is self-signed, assuming it is root
+
+Chain length: 3
+Total chain size: 4075 bytes
+Size of the biggest certificate: 1455 bytes
+```
+
+Same TPM on NovaCustom NS5x/7x and Protectli VP4630/50/70 platforms:
+
+```text
+Certificate 0 size: 1171 bytes
+Certificate 1 size: 1449 bytes
+Certificate 2 size: 1455 bytes
+Certificate is self-signed, assuming it is root
+
+Chain length: 3
+Total chain size: 4075 bytes
+Size of the biggest certificate: 1455 bytes
+```
+
+## Unknown TPM (probably also SLB 9670)
```

Comment:
@krystian-hebel TPM chips are the same on these laptops AFAIK, so let's merge it with the above

Reply:
I confirmed on the NV4X ADL model that it is the 9670.

Reply:
Moved.